lib/android/landscape.nit

```diff
@@ -12,9 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Config to set the landscape orientation
-module landscape is android_manifest_activity """
-		android:screenOrientation="landscape"
-"""
+# Lock the application in the landscape orientation
+#
+# Adds `android:screenOrientation=="sensorLandscape"` to the manifest.
+module landscape is
+	android_api_min 9
```

Comment:
It was just an informative comment as I needed to install API9 to run tests. Just stick to what you think is sane and is conform to android practices.